neuron_morphology/transforms/geometry.py

```diff
@@ -36,6 +36,35 @@ def get_vertices_from_two_lines(line1: List[Tuple], line2: List[Tuple]):
     if side1.crosses(side2):
         line2.reverse()
 
+    # validate the pia/wm does not cross the side lines
```

Comment:
This ought to be its own function. That will also give us a place to hang a warning.

Reply:
additionally, we ought to report the modified boundaries in a module output

Reply:
revised as suggested